lib/appsignal/logger.rb

```diff
@@ -109,7 +111,20 @@ def fatal(message = nil, attributes = {})
       return if FATAL < level
       message = yield if message.nil? && block_given?
       return if message.nil?
-      add(FATAL, message, @group, attributes)
+      add_with_attributes(FATAL, message, @group, attributes)
+    end
+
+    private
+
+    def add_with_attributes(severity, message, group, attributes)
+      Thread.current[:appsignal_logger_attributes] = attributes
+      add(severity, message, group)
+    ensure
+      Thread.current[:appsignal_logger_attributes]
```

Comment:
This line does nothing, probably missing something?

Reply:
No, that's correct. The `nil` assignation is missing. Fixed.